### DISCUSSION TITLE
Forward all errors and Notices to Ray ?

### DISCUSSION THREAD

**Discussion by LauGau** [spatie/wordpress-ray#34] [Q&A] [answered]
Hi ;

I just discovering the project.
I searched a bit but I am still not sure...
I wanted to see more conviniently all errors and notice of a WP website directly into Ray, so here my question :

Is there a way to forward all errors and notices to Ray ?
(Without having to call Ray()...)

Thank you for the infos ! 🙏

**Comment by freekmurze** [comment, marked as answer]
I'm not a WordPress expert on this. If there is a dedicated event that WordPress fires when handling an event we might hook into it. I'd welcome a PR that does this.

**Reply by LauGau** [reply]
Actually, I found that Sentry was working best to track "passive errors".
But for sure, I will give Ray a try to debug quickly my own code 🙂
